releasenotes-builder/src/test/java/com/github/checkstyle/internal/AbstractReleaseNotesTestSupport.java

```diff
@@ -184,7 +194,7 @@ protected Set<RevCommit> getCommits() {
      *
      * @return Copy of all current issues.
      */
-    protected Set<GHIssue> getIssues() {
+    protected static Set<GHIssue> getIssues() {
```

Comment:
protected and static is very weird fix.

what force you to do this ? can we aviod this ?
it will make more sense to make it "public static" or move it to some Util.

Reply:
This method was to allow us access to all issues incase a test needed to iterate on them.
`protected` is because this is only called from the classes that inherit this abstract class.
`static` is only being added as improvement as mentioned in first post. We don't need an instance to access this, so usually we make these things static.

I think a util will be weird as it is returning the global field from this class. It is nothing more than a simple getter.

Reply:
can we just remove "static" ?
and let it be as before -  method of Instance to not share outside details that something is static.

protected imply inheritance, static tell that all inheritence and belonging to class/object is not much relevant.

Reply:
Done.